style/rails/migration.rb

```diff
@@ -1,9 +1,10 @@
 class CreateClearanceUsers < ActiveRecord::Migration
   def change
     create_table :users  do |t|
-      t.timestamps null: false
       t.string :email, null: false
       t.string :name, null: false, default: ''
+
+      t.timestamps null: false
```

Comment:
Since we removed the guideline, should we remove this entirely, so we don't imply that timestamps should be at the bottom?